lib/internal/bootstrap/pre_execution.js

```diff
@@ -1,18 +1,51 @@
 'use strict';
 
 const {
+  Object,
   Map,
   ObjectDefineProperty,
   SafeWeakMap,
 } = primordials;
 
 const { getOptionValue } = require('internal/options');
 const { Buffer } = require('buffer');
-const { ERR_MANIFEST_ASSERT_INTEGRITY } = require('internal/errors').codes;
+const {
+  ERR_MANIFEST_ASSERT_INTEGRITY,
+  ERR_INVALID_OPT_VALUE,
+  ERR_PROTO_ACCESS,
+} = require('internal/errors').codes;
 const assert = require('internal/assert');
 
+function patchObjectProto() {
+  const disableProto = getOptionValue('--disable-proto');
+  switch (disableProto) {
+    case '':
+      // Flag was not passed
```

Comment:
Isn't this the code path taken when someone just passes `--disable-proto` without an option? If so, shouldn't we be throwing an exception in that case?

Reply:
@mscdex the options parser doesn't allow it to be passed with no argument.